iml-manager-cli/src/server.rs

```diff
@@ -373,6 +391,133 @@ pub async fn server_cli(command: ServerCommand) -> Result<(), ImlManagerCliError
 
             wait_for_cmds(cmds).await?;
         }
+        ServerCommand::Remove(config) => {
+            let term = Term::stdout();
+
+            let remove_hosts = hostlist_parser::parse(&config.hosts)?;
+
+            tracing::debug!("Parsed hosts {:?}", remove_hosts);
+
+            let (_, api_hosts) = future::try_join(get_hosts(), get_hosts()).await?;
+
+            let known_hosts = filter_known_hosts(&remove_hosts, &api_hosts.objects);
```

Comment:
This is the opposite of what we want.

We want to filter unknown hosts. This will filter known hosts.

Reply:
We also need to print a message about any hosts that were supposed to be removed, but did not exist

Reply:
Done